### ISSUE TITLE
Dumpers, Writers, Composers and Exporters

### ISSUE THREAD

**Issue by hernanmd** [pillar-markup/Microdown#359]
Multiple visitor classes have different naming conventions.
For example inspecting:

```smalltalk
#visitBold: implementors
```

It would be nice to avoid confusion to have a single naming conventions for Visitors.
Something like:

MicHTMLWriter -> MicHTMLVisitor
MicLaTeXWriter -> MicLaTeXVisitor
MicRealDumper ?? (this seems to be a code generator used for tests?)
MicRichTextComposer -> MicRichTextVisitor
MicTextualMicrodownExporter -> MicTextualVisitor

What do you think?

**Comment by kasperosterbye** [comment]
I prefer names which indicate what things do rather than what they do (visit). As I have seen composers, writers and a few others I am not sure there is an issue. 
But as I am daily struggling with trying to understand the code of others I appreciate any kind of naming scheme which is consistent, and visitor is a consistent scheme. 
However, I also believe any class that implement a `visitX:` method is most likely a visitor, and I will look at its superclass to see which kind. 

In summary - I can see the issue, but is not quite convinced it is a problem.

**Comment by kasperosterbye** [comment]
I have been looking at the examples you gave, and now I think that the class name are correctly named. There is a clear difference between a class that visits something (and produce an internal structure), and a class that produce something (based on an existing structure).
For example, the of MicRichTextComposer is that the class compose rich text, it does not visit rich text.
I suggest we close this issue.
